lib/brakeman/util.rb

```diff
@@ -124,6 +124,18 @@ def hash_access hash, key
     nil
   end
 
+  def call_has_param call, key
```

Comment:
Sorry for the late reply. That's what I tried to do:

```ruby
def call_has_param call, key
    target = arg.target
    method = target.method
    
    return call? arg and call? target and node_type? target.target, :params and method == key
  end
```

But when I run the tests I get this error:

```
ect.rb:126: void value expression (SyntaxError)
... call? arg and call? target and node_type? target.target, :p...
...                               ^
rake aborted!
Command failed with status (1): [ruby test/test.rb --pride...]
```
There are two different `node_type` methods declared in brakeman. One in `Util` and one in `Sexp`. The `Util` one returns a type, but the `Sexp` one doesn't. So I tried to enforce to use the `Util` method by doing this: `Util.node_type?` but I still get the same error.

I also moved the method `call_has_param` method to the `check_redirect` file as you suggested. I have no ideas in how to fix that. Do you have any hints about this issue?